tracing-opentelemetry/src/main/java/com/couchbase/client/tracing/opentelemetry/OpenTelemetryRequestTracer.java

```diff
@@ -33,23 +34,23 @@
 public class OpenTelemetryRequestTracer implements RequestTracer {
 
   /**
-   * Holds the actual OT tracer.
+   * Holds the actual OTel tracer.
    */
   private final Tracer tracer;
 
   /**
-   * Wraps the OpenTelemetry tracer and returns a datatype that can be passed into the requestTracer method of the
+   * Wraps OpenTelemetry and returns a datatype that can be passed into the requestTracer method of the
    * environment.
    *
-   * @param tracer the tracer to wrap.
-   * @return the wrapped tracer ready to be passed in.
+   * @param openTelemetry the OpenTelemetry instance to wrap.
+   * @return the wrapped OpenTelemetry ready to be passed in.
    */
-  public static OpenTelemetryRequestTracer wrap(final Tracer tracer) {
-    return new OpenTelemetryRequestTracer(tracer);
+  public static OpenTelemetryRequestTracer wrap(final OpenTelemetry openTelemetry) {
+    return new OpenTelemetryRequestTracer(openTelemetry);
   }
 
-  private OpenTelemetryRequestTracer(Tracer tracer) {
-    this.tracer = tracer;
+  private OpenTelemetryRequestTracer(OpenTelemetry openTelemetry) {
+    this.tracer = openTelemetry.getTracer("com.couchbase.client");
```

Comment:
Unfortunately I don't think I've seen a guidelines aside from what's here

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#get-a-tracer

Putting java here definitely makes sense, updated.

Reply:
I added `java` but now I'm wondering whether any of `jvm`, `java-core`, `java-core-io` are more appropriate since this isn't related to the `java-client` artifact

Reply:
Can you set the default to com.couchbase.client.jvm? I'm going to add subsequent options so that each higher level sdk (i.e. java, kotlin, scala) can override that "jvm" bit so we have a more specific namespace if we can get it but a sane default if not.